### PR TITLE
LPS-159413 Improve form alerts

### PR DESCRIPTION
**Pull request by sandrodw3**
cc @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/useCheckFormsValidity.js [hunk 1]
```diff
@@ -159,14 +156,19 @@ function getFormItems(layoutData) {
 	);
 }
 
-function hasUnmappedInputChild(formId, fragmentEntryLinks, layoutData) {
-	const descendantIds = getDescendantIds(layoutData, formId);
+async function checkUnmappedInputChild(
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/useCheckFormsValidity.js:159]: CI:FORWARD

Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/useCheckFormsValidity.js:159]: 🤣

Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/useCheckFormsValidity.js:159]: NO SE HABLE MÁS. Uy perdón, DON'T TALK MORE

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:forward

**Comment by ozysouza** [comment]
Hi @sandrodw3! The toggle switch to Preview Success State on Form Container it not working to display the message. Would you please have a look at?

**Comment by sandrodw3** [comment]
> Hi @sandrodw3! The toggle switch to Preview Success State on Form Container it not working to display the message. Would you please have a look at?

Hey @ozysouza! It's actually working for me. I updated some code related to that, but it's working for me locally, I don't know why the test is failing 😞 could you help? If you don't find the reason I can take a look tomorrow, thanks a lot! 🥰

**Comment by ozysouza** [comment]
> > Hi @sandrodw3! The toggle switch to Preview Success State on Form Container it not working to display the message. Would you please have a look at?
> 
> Hey @ozysouza! It's actually working for me. I updated some code related to that, but it's working for me locally, I don't know why the test is failing disappointed could you help? If you don't find the reason I can take a look tomorrow, thanks a lot! smiling_face_with_three_hearts

I had build a new bundle with the changes on this PR, I mean I've made the test locally and it was not working.

**Comment by sandrodw3** [comment]
> > > Hi @sandrodw3! The toggle switch to Preview Success State on Form Container it not working to display the message. Would you please have a look at?
> > 
> > 
> > Hey @ozysouza! It's actually working for me. I updated some code related to that, but it's working for me locally, I don't know why the test is failing disappointed could you help? If you don't find the reason I can take a look tomorrow, thanks a lot! smiling_face_with_three_hearts
> 
> I had build a new bundle with the changes on this PR, I mean I've made the test locally and it was not working. 😟
> 
>  Home.-.Liferay.DXP.-.11.August.2022.mp4

Aaaah with the form empty, got it, I was testing with some fragments inside it, thanks a lot @ozysouza 🥰

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:stop

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by ozysouza** [comment]
ci: forward

**Comment by ozysouza** [comment]
ci:forward

**Pull request closed without merging**